typescript/.changeset/config.json

```diff
@@ -7,5 +7,8 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["@coinbase/*-example"]
+  "ignore": ["@coinbase/*-example"],
+  "___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH": {
+    "onlyUpdatePeerDependentsWhenOutOfRange": true
+  }
```

Comment:
I noticed `viem` uses this. [See here](https://github.com/wevm/viem/blob/main/.changeset/config.json#L13-L15)

This makes it so that `changesets` doesn't bump dependencies on peerDependencies. So in this case, `@coinbase/vercel-ai-sdk` won't be bumped when `@coinbase/agentkit` is bumped (until it goes out of range, which currently it never will until we get to `v1.0.0`.